assets/component-predictive-search.css

```diff
@@ -14,7 +14,8 @@
 }
 
 .predictive-search--search-template {
-  z-index: 2
+  z-index: 2;
+  width: 100%;
```

Comment:
I think we either need the width calculated the same way as for `predictive-search` so with `calc(100% + 0.2rem)` or we could cancel the left positioning to be `left: 0;` in this case 🤔 

![](https://screenshot.click/28-33-v73f3-lei8q.png)